tests/Data/Formatter/XmlDataResponseFormatterTest.php

```diff
@@ -39,4 +39,111 @@ public function testFormatterEncoding(): void
         );
         $this->assertSame(['application/xml; ISO-8859-1'], $result->getHeader('Content-Type'));
     }
+
+    public function testFormatterVersion(): void
+    {
+        $factory = new Psr17Factory();
+        $dataResponse = new DataResponse('test', 200, '', $factory);
+        $formatter = new XmlDataResponseFormatter();
+        $formatter = $formatter->withVersion('1.1');
+        $result = $formatter->format($dataResponse);
+        $result->getBody()->rewind();
+
+        $this->assertSame(
+            "<?xml version=\"1.1\" encoding=\"UTF-8\"?>\n<response>test</response>\n",
+            $result->getBody()->getContents()
+        );
+        $this->assertSame(['application/xml; UTF-8'], $result->getHeader('Content-Type'));
+    }
+
+    public function testFormatterRootTag(): void
+    {
+        $factory = new Psr17Factory();
```

Comment:
Please use mock instead

Reply:
Why? We're testing output so we need a working stream to capture it.